app/views/users/index.html.haml

```diff
@@ -1,7 +1,12 @@
+%csrf_meta_tag
 .page-header
-  %h1
-    Users
-    = link_to 'New User', new_user_path, :class => 'btn btn-success pull-right'
+  %div{:class => 'header-div'}
+    %h1
+      Users
+    %div{:class => 'header-right'}
+      .btn.btn-primary.pull-right{"data-toggle" => "popover", "title"=>"Quick Info" } Import Users      
+      = link_to 'New User', new_user_path, :class => 'btn btn-success pull-right'
+
```

Comment:
This part is for a different feature: letting users upload a csv file to create new users